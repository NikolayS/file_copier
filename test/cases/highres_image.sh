#!/bin/bash

# to run this scenario, COPIERHOST should be defined (`export COPIERHOST=http://your.copier.host.name`)

http GET "http://$COPIERHOST/?wh=1&md5=1&src=http://mars.nasa.gov/msl/imgs/2015/10/mars-curiosity-rover-msl-big-sky-selfie-portrait-pia19920-full.jpg" -h | grep -v "Date:"

