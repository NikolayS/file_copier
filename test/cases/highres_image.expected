HTTP/1.1 200 OK
Server: nginx
Content-Type: text/html; charset=UTF-8
Transfer-Encoding: chunked
Connection: keep-alive
X-File-Copier-Md5: 469b2839fa8bfbb0f9349595cf918494
X-File-Copier-Img-Width: 7149
X-File-Copier-Img-Height: 10036
X-File-Copier-Size: 13144829
X-Location: /data/2f/01/9e/8f/2f019e8f295ddc6418707839aeb49c5db123734950a508f76452aac1abb632a0.jpg
Content-Encoding: gzip

